mix.exs

```diff
@@ -24,12 +24,12 @@ defmodule Jwt.Mixfile do
 
   defp deps do
     [
-    {:httpoison, "~> 0.9.0" },
+    {:httpoison, "~> 0.13" },
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.